packages/grafana-ui/src/components/GraphNG/types.ts

```diff
@@ -0,0 +1,20 @@
+import { DataFrameFieldIndex } from '@grafana/data';
+
+/**
+ * Mode to describe if a legend is isolated/selected or being appended to an existing
+ * series selection.
+ * @public
+ */
+export enum GraphNGLegendEventMode {
+  toggleSelection = 'select',
+  appendToSelection = 'append',
```

Comment:
For enums we usually use PascalCase

Reply:
Looks like we mix a lot in the code base. But I will change this one. Thanks for the feedback.